### PR TITLE
[#2902] LSP4J 0.21.2

### PR DESCRIPTION
**Pull request by cdietrich**

### CODE REVIEW

File: org.eclipse.xtend.core/src/org/eclipse/xtend/core/compiler/XtendCompiler.java [hunk 1]
```diff
@@ -54,7 +54,6 @@
 import org.eclipse.xtext.xbase.XbasePackage;
 import org.eclipse.xtext.xbase.compiler.GeneratorConfig;
 import org.eclipse.xtext.xbase.compiler.IGeneratorConfigProvider;
-import org.eclipse.xtext.xbase.compiler.Later;
```
Comment (cdietrich) [inline org.eclipse.xtend.core/src/org/eclipse/xtend/core/compiler/XtendCompiler.java:57]: why is this changed ....

**Comment by LorenzoBettini** [comment]
The build failure is related to the issue that's already solved on main

**Comment by cdietrich** [comment]
Have rebased

File: org.eclipse.xtext.ide.tests/META-INF/MANIFEST.MF [hunk 2]
```diff
@@ -14,8 +14,8 @@ Require-Bundle: org.eclipse.xtext;bundle-version="2.34.0";visibility:=reexport,
  org.eclipse.core.runtime;bundle-version="3.24.100",
  org.eclipse.xtext.ide;bundle-version="2.34.0",
  org.junit;bundle-version="4.13.2",
- org.eclipse.lsp4j;bundle-version="[0.21.1,0.22.0)";resolution:=optional,
- org.eclipse.lsp4j.jsonrpc;bundle-version="[0.21.1,0.22.0)";resolution:=optional,
+ org.eclipse.lsp4j;bundle-version="[0.21.2,0.22.0)";resolution:=optional,
```
Comment (szarnekow) [inline org.eclipse.xtext.ide.tests/META-INF/MANIFEST.MF:17]: Should we add version constraints to xtext.testing, too?

Comment (cdietrich) [reply org.eclipse.xtext.ide.tests/META-INF/MANIFEST.MF:17]: good question

Comment (cdietrich) [reply org.eclipse.xtext.ide.tests/META-INF/MANIFEST.MF:17]: dont know why we havent

Comment (szarnekow) [reply org.eclipse.xtext.ide.tests/META-INF/MANIFEST.MF:17]: I'd assume it's an oversight.

**Review by szarnekow — APPROVED** [review]

**Review by szarnekow — APPROVED** [review]
LGTM

**Comment by LorenzoBettini** [comment]
@cdietrich if you merge it I can rebase https://github.com/eclipse/xtext/pull/2903

**Comment by cdietrich** [comment]
Can you merge

**Comment by szarnekow** [comment]
> Can you merge

Done.
